test/e2e/README.md

```diff
@@ -0,0 +1,14 @@
+# Running e2e tests for smi-adapter-istio
+
+## Prerequistes
+- Running Kubernetes cluster
+- [Install Istio](https://github.com/deislabs/smi-adapter-istio/tree/master/docs/smi-trafficsplit#install-istio) on Kubernetes cluster
+- [Install operator-sdk cli](https://github.com/operator-framework/operator-sdk/blob/master/doc/user-guide.md#install-the-operator-sdk-cli)
+
+## Run e2e tests
+
+```console
+$ cd $GOPATH/src/github.com/deislabs/smi-adapter-istio/
+
+$ operator-sdk test local ./test/e2e --namespaced-manifest test/testdata/operator-and-rbac.yaml --namespace istio-system
```

Comment:
yea. I'll add that as a make target. great idea.

It looks like you're running this on a cluster where the smi-adapter-istio manifests have been installed before. You'll want to clean all that up by running `kubectl delete -f deploy/` in root so that this will work as expected.

Reply:
This is *probably* out of scope, but since I've been doing it everywhere else. WDYT about using `kind` for this so that the cluster is guaranteed to be vanilla.

Reply:
Also, just retested and everything passed.

Reply:
yea I definitely like the idea of using `kind` for this stuff. I'll add an issue for it.